care/utils/assetintegration/onvif.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, meta):
             self.meta = json.loads(meta)
             self.meta = meta
             self.name = meta["camera_type"]
-            self.host = meta["camera_address"]
+            self.host = meta["local_ip_address"]
```

Comment:
Is this change in onvif intended?

Reply:
yup, `camera_address` was not an accurate name and has been replaced in [#771](https://github.com/coronasafe/care/pull/771).